laravel/gitlab/index.html

```diff
@@ -118,7 +118,7 @@ <h3>Gitlab CI configuration for a Laravel application running with Kool.dev</h3>
   php-security-checker: kool run phpqa security-checker security:check composer.lock
   phpunit: kool exec app ./vendor/bin/phpunit
   start:chrome: kool docker -- --name="chromedriver" --network=$KOOL_GLOBAL_NETWORK -d --shm-size=256M fireworkweb/chromedriver:latest
-  stop:chrome: docker stop chromedriver
+  stop:chrome: kool docker stop chromedriver
```

Comment:
@dbpolito @fabriciojs - in particular, confirm `kool` was missing here.

Reply:
@indiegenus actually here we cannot use `kool`. `kool docker` is an abstraction straight to `docker run` so we cannot use for `docker stop`, like in this case. So we need to keep using `docker` bin directly.